docker/Dockerfile

```diff
@@ -86,10 +86,11 @@ RUN \
 	# Remove Git and clean apt cache
 	apt-get clean && \
 	apt-get remove -y git && \
-	apt-get -y autoremove
+	apt-get -y autoremove && \
+	chmod +x docker/entrypoint.sh
```

Comment:
Same as https://github.com/telefonicaid/iotagent-json/pull/380#discussion_r258422051

Reply:
Response same as https://github.com/telefonicaid/iotagent-json/pull/380#discussion_r258503643 :smile: 

I believe this is necessary - more transparent, secure and easier to maintain

Reply:
NTC according to https://github.com/telefonicaid/iotagent-json/pull/380#discussion_r259465167